### PR TITLE
feat: [DEL-3179]: expose UI apis for ng delegate token

### PR DESCRIPTION
**Pull request by arpit3107**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/30293)

### CODE REVIEW

File: 420-delegate-service/src/main/java/io/harness/delegate/resources/DelegateNgTokenResource.java [hunk 1]
```diff
@@ -129,10 +125,8 @@ public DelegateNgTokenResource(DelegateNgTokenService delegateTokenService) {
   public RestResponse<List<DelegateTokenDetails>>
   getDelegateTokens(@Parameter(description = NGCommonEntityConstants.ACCOUNT_PARAM_MESSAGE) @QueryParam(
                         "accountId") @NotNull String accountId,
-      @Parameter(description = NGCommonEntityConstants.ORG_PARAM_MESSAGE) @QueryParam(
-          NGCommonEntityConstants.ORG_KEY) String orgId,
-      @Parameter(description = NGCommonEntityConstants.PROJECT_PARAM_MESSAGE) @QueryParam(
-          NGCommonEntityConstants.PROJECT_KEY) String projectId,
+      @Parameter(description = NGCommonEntityConstants.ORG_PARAM_MESSAGE) @QueryParam("orgId") String orgId,
+      @Parameter(description = NGCommonEntityConstants.PROJECT_PARAM_MESSAGE) @QueryParam("projectId") String projectId,
```
Comment (abhinav-harness) [inline 420-delegate-service/src/main/java/io/harness/delegate/resources/DelegateNgTokenResource.java:129]: use constant in query param string.

File: 400-rest/src/main/java/software/wings/resources/DelegateSetupResourceV3.java [hunk 2]
```diff
@@ -1118,11 +1068,11 @@ public void setTags(List<String> tags) {
       })
   public RestResponse<DelegateGroupListing>
   list(@Parameter(description = NGCommonEntityConstants.ACCOUNT_PARAM_MESSAGE) @NotEmpty @QueryParam(
-           NGCommonEntityConstants.ACCOUNT_KEY) @AccountIdentifier String accountIdentifier,
+           "accountId") @AccountIdentifier String accountIdentifier,
       @Parameter(description = NGCommonEntityConstants.ORG_PARAM_MESSAGE) @QueryParam(
```
Comment (abhinav-harness) [inline 400-rest/src/main/java/software/wings/resources/DelegateSetupResourceV3.java:1072]: why the change?

**Review by abhinav-harness — CHANGES_REQUESTED** [review]

**Review by raghuAtWings — APPROVED** [review]
> Reviewed 2 of 4 files at r1, 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30293)* status: :shipit: complete! ✓

**Review by raghuAtWings — APPROVED** [review]
> Reviewed 4 of 4 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30293)* status: :shipit: complete! ✓

**Review by abhinav-harness — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30293)* status: 2 file(s) to review.

**Review by abhinav-harness — APPROVED** [review]
> Reviewed 1 of 3 files at r2, 4 of 4 files at r3, 2 of 2 files at r4.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30293)* status: :shipit: complete! ✓

**Pull request merged**